### PR TITLE
fix:translate configure-webhook.md

### PR DESCRIPTION
**Pull request by 123liubao**
Signed-off-by: 123liubao <chenly_61@163.com>

Description：

The webhook configuration doc is missing a Chinese version.

Fix :  #2058

### CODE REVIEW

File: content/zh/docs/cluster-administration/platform-settings/notification-management/configure-webhook.md [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
 
-9. Click **OK** after you finish.
+9. 完成后点击 **确定** 。
 
    {{< notice note >}}
 
-   - After the notification conditions are set, the recepients will receive only notifications that meet the conditions.
-   - If you change the existing configuration, you must click **OK** to apply it.
+   - 设置通知条件后，接收方只会收到满足条件的通知。
+   - 如果更改现有配置，则必须单击 **确定** 才能应用修改后的配置。
```
Comment (Felixnoo) [inline content/zh/docs/cluster-administration/platform-settings/notification-management/configure-webhook.md:61]: `单击` -> `点击`

**Review by Felixnoo — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution. I would like to request one change.

File: content/zh/docs/cluster-administration/platform-settings/notification-management/configure-webhook.md (hunk 1 above)
Comment (123liubao) [reply content/zh/docs/cluster-administration/platform-settings/notification-management/configure-webhook.md]: Done.

**Comment by 123liubao** [comment]
> Hi, thanks for your contribution. I would like to request one change.

Thank you for your advice, I have modified the document.

**Pull request merged**